dds/DCPS/DataReaderImpl_T.h

```diff
@@ -915,6 +915,26 @@ namespace OpenDDS {
       if (!encap.to_encoding(encoding, MarshalTraitsType::extensibility())) {
         return;
       }
+
+      if (decoding_modes_.find(encoding.kind()) == decoding_modes_.end()) {
+        if (DCPS_debug_level >= 8) {
+          ACE_ERROR((LM_WARNING, ACE_TEXT("(%P|%t) WARNING ")
```

Comment:
```suggestion
        if (DCPS_debug_level >= 1) {
          ACE_DEBUG((LM_WARNING, ACE_TEXT("(%P|%t) WARNING ")
```

Warnings should use `ACE_DEBUG` as a matter of convention and log level one because they are urgent.